tests/util.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+from git import Actor, Repo
+
+SOME_COMMIT_MESSAGE = "some-commit-message"
+SOME_ACTOR = Actor("some-user", "some-user@wayfair.com")
```

Comment:
this is a really big nitpick, but I think there are benefits to leaving `wayfair` out of actual code. we currently reference `wayfair` throughout the repo in various configurations and text, but not in code.

feel free to ignore this if you dont see the point!

```suggestion
SOME_ACTOR = Actor("some-user", "some-user@some-company.com")
```